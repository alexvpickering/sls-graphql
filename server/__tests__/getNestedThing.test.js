const axios = require("axios");
const gql = require("graphql-tag");

it("gets a nested thing", () => {
  const variables = { id: "1" };
  const query = gql`
    query GetNestedThing($id: ID!) {
      getNestedThing(id: $id) {
        id
        things {
          id
          one
          two
        }
      }
    }
  `;

  return axios
    .post("http://localhost:4000/graphql", {
      query,
      variables
    })
    .then(res => {
      expect(res.status).toBe(200);
      expect(res.data.data.getNestedThing).toMatchSnapshot();
    });
});
