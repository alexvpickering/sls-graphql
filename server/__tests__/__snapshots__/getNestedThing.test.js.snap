// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gets a nested thing 1`] = `
Object {
  "id": "1",
  "things": Array [
    Object {
      "id": "1",
      "one": "thing 1",
      "two": "thing 2",
    },
    Object {
      "id": "2",
      "one": "thing 1 two",
      "two": "thing 2 two",
    },
  ],
}
`;
